### DISCUSSION TITLE
Using FontWeights with TypeScript

### DISCUSSION THREAD

**Discussion by ademkikaj** [microsoft/fluentui#17570] [Q&A] [answered]
I am having troubles assigning `FontWeights` to `fontWeight` style using TypeScript. 
The error that I receive is `Type 'IFontWeight' is not assignable to type 'FontWeightProperty'.`

I did a workaround, but I don't feel comfortable with that. I believe that some conversion needs to be done automatically when using FontWeights to fontWeight.

For now I am making an explicit conversation:

`<span style={{ fontSize: FontSizes.size32, fontWeight: FontWeights.light as number }}>Span Text</span>;`

Any help or suggestion is appreciated.

**Comment by blendsdk** [comment, marked as answer]
I use the same trick for the time being.

**Reply by JustSlone** [reply]
Yep, this is because the IFontWeight definition doesn't match the one expected by the style attribute. It looks like IFontWeight allows numbers as strings ('100', '200') whereas FontWeightProperty does not. This would be a breaking change to adjust right now unfortunately.
